other/materials_designer/create_interface_with_min_strain_zsl.ipynb

```diff
@@ -11,7 +11,7 @@
     "<h2 style=\"color:green\">Usage</h2>\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        distance_z=3.0,                  # distance between two planes, in Angstroms
</pre><p>distance between substrate and layer</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>